cli/app.go

```diff
@@ -61,16 +65,54 @@ Examples:
 `)
 }
 
+func promptYesNo(msg string) (result bool) {
+	fmt.Print(msg)
+	fmt.Print(" (yes/no): ")
+	for {
+		var answer string
+		fmt.Scanln(&answer)
+		switch answer {
+		case "y", "yes":
+			return true
+		case "n", "no":
+			return false
+		default:
+			fmt.Print("Please type 'yes' or 'no': ")
+		}
+	}
+}
+
 func runCreate(args *docopt.Args, client *controller.Client) error {
 	app := &ct.App{}
 	app.Name = args.String["<name>"]
+	remote := args.String["<remote>"]
+
+	// Test if remote name exists and prompt user
+	remotes, err := gitRemoteNames()
+	if err != nil {
+		return err
+	}
+
+	if !args.Bool["--yes"] {
+		for _, r := range remotes {
+			if r == remote {
+				fmt.Println("There is already a git remote called", remote)
+				if !promptYesNo("Are you sure you want to replace it?") {
+					log.Println("The app was not created. Please, declare the desired local git remote name with --remote flag.")
+					return nil
+				}
+			}
+		}
+	}
 
+	// Create the app
 	if err := client.CreateApp(app); err != nil {
 		return err
 	}
 
-	exec.Command("git", "remote", "remove", "flynn").Run()
-	exec.Command("git", "remote", "add", "flynn", gitURLPre(clusterConf.GitHost)+app.Name+gitURLSuf).Run()
+	// Register git remote
+	exec.Command("git", "remote", "remove", remote).Run()
+	exec.Command("git", "remote", "add", remote, gitURLPre(clusterConf.GitHost)+app.Name+gitURLSuf).Run()
```

Comment:
Ah, good point!

In this case, the command will need more options, to specify container images and other methods in future, right?